crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -628,14 +628,39 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsRequest> for Pa
             .as_ref()
             .map(|payment_method_billing| payment_method_billing.address_id.clone());
 
+        // If processor_payment_token is passed in request then populating the same in PaymentData
```

Comment:
we do not need this in payments confirm

Reply:
As discussed, For future usecases we will let this flow be present in confirm as well.